Source/SourceKittenFramework/Xcode.swift

```diff
@@ -50,45 +50,10 @@ internal enum XcodeBuild {
      - returns: `xcodebuild`'s STDOUT output and, optionally, both STDERR+STDOUT output combined.
      */
     internal static func launch(arguments: [String], inPath path: String, pipingStandardError: Bool = true) -> Data {
-        let task = Process()
-        let pathOfXcodebuild = "/usr/bin/xcodebuild"
-        task.arguments = arguments
-
-        let pipe = Pipe()
-        task.standardOutput = pipe
-
-        if pipingStandardError {
-            task.standardError = pipe
-        }
-
-        do {
-        #if canImport(Darwin)
-            if #available(macOS 10.13, *) {
-                task.executableURL = URL(fileURLWithPath: pathOfXcodebuild)
-                task.currentDirectoryURL = URL(fileURLWithPath: path)
-                try task.run()
-            } else {
-                task.launchPath = pathOfXcodebuild
-                task.currentDirectoryPath = path
-                task.launch()
-            }
-        #elseif compiler(>=5)
-            task.executableURL = URL(fileURLWithPath: pathOfXcodebuild)
-            task.currentDirectoryURL = URL(fileURLWithPath: path)
-            try task.run()
-        #else
-            task.launchPath = pathOfXcodebuild
-            task.currentDirectoryPath = path
-            task.launch()
-        #endif
-        } catch {
-            return Data()
-        }
-
-        let file = pipe.fileHandleForReading
-        defer { file.closeFile() }
```

Comment:
The [docs](https://developer.apple.com/documentation/foundation/pipe/1414352-filehandleforreading) say this isn't necessary so I left it out.